.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: ruby
 cache: bundler
 rvm:
-- 1.9.3
-- 2.0.0
-- 2.1
-- 2.2
+  - 1.9.3
+  - 2.0.0
+  - 2.1
+  - 2.2
+  - jruby-head
```

Comment:
You're right, its probably best to lock to some versions. I'll add the first version we support (`jruby-9.0.0.0`) and the latest (`jruby-9.0.4.0`) instead.